install/aws/dev/docker/php-fpm/etc/php/99-ffi.ini

```diff
@@ -0,0 +1 @@
+ffi.enable=true
```

Comment:
What does the 99 in the filename mean?

Reply:
[It's a UNIX convention](https://unix.stackexchange.com/questions/134520/what-is-the-number-prefix-in-config-files-from-d-directory). We probably don't need numbering but I'm doing it for consistency.